src/apyfixed_util.cc

```diff
@@ -27,12 +27,18 @@ mp_limb_t get_data_from_double(double value, int bits, int frac_bits, int shift_
     // correct position
     auto left_shift_amnt = exp + frac_bits - 52 - 1023;
     if (left_shift_amnt >= 0) {
-        man <<= left_shift_amnt;
+        if (left_shift_amnt < 64) {
```

Comment:
It would be good if this was tested. Not so obvious what the behavior actually is here when it comes to overflow etc.